corehq/apps/locations/models.py

```diff
@@ -819,6 +819,7 @@ class LocationRelation(models.Model):
         SQLLocation, on_delete=models.CASCADE, related_name="+", to_field='location_id')
     location_b = models.ForeignKey(
         SQLLocation, on_delete=models.CASCADE, related_name="+", to_field='location_id')
+    distance = models.PositiveSmallIntegerField(null=True)
```

Comment:
Is there an expected unit of measurement here?  Even just as a comment it could be helpful.

Reply:
There is no unit of measurement, this is purely meant as a relative "distance" that could vary from project to project. I was using the term "distance" as that's what it's called for graphs (which is basically what this structure is), but understand that could be confusing when talking about physical locations. any ideas on how to make that more clear?

Reply:
No, that seems fine.  Was mostly curious if this was meant to relate at all to lat/long (which is stored on the location)